README.md

```diff
@@ -2,6 +2,18 @@
 
 Kaguya is a ChatGPT plugin that allows you to load and edit your local files in a controlled way, as well as run any Python, JavaScript, and bash script. This makes it a powerful tool for developers, enabling them to interact with their file system and run scripts directly from ChatGPT.
 
+## More About Kaguya
+
+Check out the first demo video of Kaguya: 
+
+https://github.com/pushkaraj2007/kaguya/assets/107422421/3000d44a-9c89-4c0f-99bf-8df131aa71ee
+
+Check out the second demo video of Kaguya:
+
+https://github.com/pushkaraj2007/kaguya/assets/107422421/6b180222-6ac5-4d42-9b40-7d48fd37f961
```

Comment:
can we actually swap the order and show the second video first? I think it'll be more impactful that way.

also can we reduce the repetitiveness in the wording of these two sentences?